DataConnectors/AWS-S3/Utils/HelperFunctions.ps1

```diff
@@ -28,12 +28,15 @@ function Write-RequiredConnectorDefinitionInfo
         Specifies the connector destination table
     #>
     param(
-        [Parameter(Mandatory=$true,Position=0)][string]$DestinationTable
+        [Parameter(Mandatory=$false,Position=0)][string]$DestinationTable
```

Comment:
why it is not mandatory? is it not required for custom logs?

Reply:
No, we don't know the destination table and I don't want to ask the customer for it (it's customer content)